tests/memex/storage_test.py

```diff
@@ -407,10 +434,10 @@ def fetch_annotation(patch):
 
 
 @pytest.fixture
-def models(patch):
-    models = patch('memex.storage.models', autospec=False)
-    models.Annotation.return_value.is_reply = False
-    return models
+def patched_annotation(patch):
+    klass = patch('memex.storage.models.Annotation')
+    klass.return_value.is_reply = False
+    return klass
```

Comment:
I like the method you describe in the diff, I replaced my commit with this one :+1: